Sources/App/Settings/SettingsDetailViewController.swift

```diff
@@ -454,7 +453,29 @@ class SettingsDetailViewController: HAFormViewController, TypedRowControllerType
                         $0.disabled = true
                     },
                 ], getter: {
-                    Self.getSceneRows($0)
+                    var baseRows: [BaseRow] = []
+                    if $0 == scenes.first {
+                        let toggleAllSwitch = SwitchRow()
+                        toggleAllSwitch.title = NSLocalizedString(
+                            "Select All",
```

Comment:
We don't run genstrings at all, you need to modify Localizable.strings and use the Swiftgen-generated L10n module to access them with compile time safety.

Reply:
Got it, I've added a new English string for it then